### ISSUE TITLE
Prompting users instead of specifying path arguments

### ISSUE THREAD

**Issue by augbog** [ryanburgess/sassysass#3]
Right now if you want to create custom modules and/or pages, the user has to input

```
sassysass-page name=[filename] path=[path to Sass directory]
```

This is fine, but I think if we could simply prompt the user similar to how when you do `npm init` to make a `package.json` it will go line-by-line what things the user can fill out. This would also help when users want to enter extra information including description, etc. If a user enters nothing, it will go to the default.

![screen shot 2015-05-24 at 1 50 38 pm](https://cloud.githubusercontent.com/assets/2054339/7789588/e2994986-021b-11e5-9430-076b16129234.png)

I'm looking into this and possibly leveraging [prompt](https://github.com/flatiron/prompt) to help do this.

**Comment by ryanburgess** [comment]
That's a great idea! That's a lot nicer than having to pass query parameters. I've implemented the prompt library for both the module and page script. Have a look and let me know what you think.

**Comment by augbog** [comment]
Awesome! Just tried it :) Way cleaner now! Closing because this has now been addressed in 6db6cf6.
